templates/todomvc/src/main.js

```diff
@@ -1,11 +1,77 @@
+let near = null;
+
+async function init() {
```

Comment:
boilerplate code needs to live in Nearlib, not in `main.js`

Reply:
I am not a big fan of putting this in nearlib. How about some other file specific to nearstudio?

Reply:
imo, this should just go to browser nearlib only (i.e. `exports.js`).